### DISCUSSION TITLE
Valoracion de Practica

### DISCUSSION THREAD

**Discussion by Gona79** [ISPC-TST-Sensores-y-Actuadores/practica-de-la-1er-semana-grupo-1#16] [Announcements]
Nota grupal: 7,8 (Ocho)

Leonardo gonzalez  Calefactor IoT: 7(Siete)
Maria Carolina Niz; Baston para no videntes y otros  : 9(nueve)
Pedro Omar Rojo; Lavarropa automatico IoT: 7(siete)
Fernando Vexenat; Pañal Inteligente y otros: 9 (nueve)
Miguel Angel; Tablero alumbrado publico: 7(Siete)

***

**Daniela Mazzini: N/C   No realiza**

**Comment by Daninja111** [comment]
@Gona79  Profe, acabo de agregar entrega de la 1era semana, lamentablemente me confundí con la fecha límite, había entendido que era hasta el viernes de esta semana y organicé la entrega de esa forma, le daré mayor prioridad las proximas veces para no estar en esta situación. Agradecería pueda echarle un vistazo de igual forma, gracias!
https://github.com/ISPC-TST-Sensores-y-Actuadores/practica-de-la-1er-semana-grupo-1/blob/7e1af71b27c513ef889e956ae4459b81b38ed16e/Semana-1-Practica-1/Silla_ergo_IoT.pdf
